make_statistic_plots.py

```diff
@@ -0,0 +1,582 @@
+#Author: Nicole Schanche
+#Updated Oct 12, 2022
+#This script makes the TESS statistics plots for tess-website-lite. This is very much a work in progress
+
+import pandas as pd
+import sqlite3
+import json
+import re
+import numpy as np
+#import geograpy
+import itertools
+import plotly.graph_objects as go
+import plotly.express as px
+from datetime import date, datetime
+from dateutil import rrule
+import matplotlib.pyplot as plt
+from wordcloud import WordCloud
+from collections import defaultdict
+from collections import Counter
+
+
+# Get all of the data you will need to make the TESS statistics plots later on
+
+# Read in the latest database file from tpub (Note that this is static for now, but can be adjusted later when we decide how we want to get tpub data for the website. 
+query = sqlite3.connect('content/data/tpub.db').execute('SELECT * FROM pubs')
+
+cols = [column[0] for column in query. description]
+results = pd.DataFrame.from_records(data = query.fetchall(), columns = cols)
+results = results[results['mission'] == 'tess'].reset_index(drop=True)
+
+
+# We will want to pull in some information on the author's affiliation (country and state (if applicable))
+# CSV from Christina
+countries = pd.read_csv("content/data/countries_codes_and_coordinates.csv")
+for column in countries.columns[1:]:
+    countries[column] = [c.strip().replace('"', "") for c in countries[column]]
+countries['Country'] = countries['Country'].replace('Jersey','Bailiwick of Jersey') #Getting confused with New Jersey
```

Comment:
```suggestion
countries['Country'].replace('Jersey','Bailiwick of Jersey', inplace=True)
```

Easier to read? Don't feel strong about this

Reply:
Either way is fine for me. I changed it to be consistent with your coding style